README.md

```diff
@@ -1,2 +1,22 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+A Node.js CLI tool for searching for, or opening a YouTube video with [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io). To use the zap-cli command-line tool, clone the repository and run a global npm install inside of it.
+
+```
+git clone https://github.com/zen-audio-player/zap-cli && cd zap-cli
+npm install -g
+```
+
+You can also access the zap command-line tool via the [zap-cli npm package](
+https://www.npmjs.com/package/zap-cli). In order to use the zap-cli package globally, you must change into the directory of the package where you will then need to install it globally. Below is an example of installing zap-cli via npm.
+
+```
+npm install zap-cli 
+zap-cli@1.0.0 /usr/local/lib/node_modules/zap-cli
+└── open@0.0.5
+cd /usr/local/lib/node_modules/zap-cli
+npm install -g
+```
+
+Once done, you will then have access to the `zap` command globally throughout your computer. Below, you can see the zap-cli in action.
+
+![Zap command line tool Example](http://g.recordit.co/KXdK5O5sPJ.gif)
```

Comment:
Can we use [this](https://www.youtube.com/watch?v=koJv-j1usoI) as the demo for consistency?